docs/api.rst

```diff
@@ -1348,6 +1348,59 @@ to handle it, which defaults to print a traceback and ignoring the exception.
     :param invite: The invite that was deleted.
     :type invite: :class:`Invite`
 
+.. function:: on_automod_rule_create(rule)
+
+    Called when an :class:`AutoModRule` is created.
+    You must have the :attr:`~Permissions.manage_guild` permission to receive this.
+
+    This requires :attr:`Intents.automod_configuration` to be enabled.
+
+    .. versionadded:: 2.6
+
+    :param rule: The auto moderation rule that was created.
+    :type rule: :class:`AutoModRule`
+
+.. function:: on_automod_rule_update(rule)
+
+    Called when an :class:`AutoModRule` is updated.
+    You must have the :attr:`~Permissions.manage_guild` permission to receive this.
+
+    This requires :attr:`Intents.automod_configuration` to be enabled.
+
+    .. versionadded:: 2.6
+
+    :param rule: The auto moderation rule that was updated.
+    :type rule: :class:`AutoModRule`
+
+.. function:: on_automod_rule_delete(rule)
+
+    Called when an :class:`AutoModRule` is deleted.
+    You must have the :attr:`~Permissions.manage_guild` permission to receive this.
+
+    This requires :attr:`Intents.automod_configuration` to be enabled.
+
+    .. versionadded:: 2.6
+
+    :param rule: The auto moderation rule that was deleted.
+    :type rule: :class:`AutoModRule`
+
+.. function:: on_automod_action_execution(execution)
+
+    Called when an auto moderation action is executed due to a rule triggering for a particular event.
+    You must have the :attr:`~Permissions.manage_guild` permission to receive this.
+
+    The guild this action has taken place in can be accessed using :attr:`AutoModActionExecution.guild`.
+
+    This requires :attr:`Intents.automod_execution` to be enabled.
+
+    In addition, :attr:`Intents.message_content` must be enabled to receive non-empty values
+    for :attr:`AutoModActionExecution.content` and :attr:`AutoModActionExecution.matched_content`.
```

Comment:
One *very* important thing to note in a nice, bold admonition, this event can fire **multiple** times for the same rule and trigger. The reason for this is it is sent for *each* action taken, rather than one that encompasses all of the rules. This should be noted, possibly with a warning, so devs are aware of this and don't triple warn a person for a single action.

Reply:
https://github.com/DisnakeDev/disnake/pull/530/commits/0a6e40c2823e6b69e60387f4bc070d4004471b8a